src/components/images_uploading/DisplayImageList.js

```diff
@@ -63,11 +91,30 @@ class DisplayImageList extends Component {
               <UserAssignmentDropDown imageId={image._id} users={users} />
 
               <Button
+                style={{ marginLeft: 3 }}
                 variant="success"
                 href={`${process.env.REACT_APP_BASEURL}/images/${image.s3key}`}
               >
+                {/* this one can't be changed to Link to. Btw, why is 'link to' better than href? 'Link to' wil attach url prefix 'http://localhost:3000' to the link you set up. For example, if I change the code below to link to, the url will become 'http://localhost:3000/http://localhost:5000/images/07cea77f-a2ff-422f-ab35-3e918a025262.jpeg', which is not the route we want to have to show image.

```

Comment:
See my comment in code here: This one can't be changed to 'Link to'

Reply:
That makes sense for now. When the image editing component is ready we can replace this direct link to backend with a Link to image editing route.

Reply:
Got it! Thanks :)